algebra-core/src/fields/mod.rs

```diff
@@ -191,50 +213,99 @@ pub trait FpParameters: 'static + Send + Sync + Sized {
     /// (Should equal `SELF::MODULUS_BITS - 1`)
     const CAPACITY: u32;
 
-    /// 2^s * t = MODULUS - 1 with t odd. This is the two-adicity of
-    /// `Self::MODULUS`.
-    const TWO_ADICITY: u32;
-
     /// t for 2^s * t = MODULUS - 1
     const T: Self::BigInt;
 
-    /// 2^s root of unity computed by GENERATOR^t
-    const ROOT_OF_UNITY: Self::BigInt;
-
     /// (t - 1) / 2
     const T_MINUS_ONE_DIV_TWO: Self::BigInt;
```

Comment:
Should we move `T` and `T_MINUS_ONE_DIV_TWO` also to `FftParameters`?

Reply:
I thought about that, too. But then, they are currently not used in any FFT context as far as I could see.

`T_MINUS_ONE_DIV_TWO` is only used in the sqrt implementation and I couldn't find any usage of `T` at all.

Reply:
Alright, that's fair. We can always move it in the future if the need arises. Thanks!